cli/main.go

```diff
@@ -128,6 +157,76 @@ func main() {
 		EnableBashCompletion:   true,
 		UseShortOptionHandling: true,
 		Commands: []*cli.Command{
+			{
+				Name:  "stale-balance",
+				Args:  true,
+				Usage: "If needed, calls `verifyStaleBalances` to correct the balance on a pod you don't own. This will attempt to (1) conclude any existing checkpoint on the pod, and then (2) invoke verifyStaleBalances() to start another, more up-to-date, checkpoint.",
```

Comment:
Note that anything surrounded by backticks (eg \`verifyStaleBalances\` above) gets added as a "variable name"/"target" in the CLI help text. Would recommend not using these unless you're trying to change the help text

Run `./cli -h` to see what i'm talking about